profiler/src/main/java/com/navercorp/pinpoint/profiler/instrument/PlainClassLoaderHandler.java

```diff
@@ -102,7 +102,9 @@ private ClassPool createClassPool(ClassLoader classLoader) throws NotFoundExcept
     private Class<?> injectClass0(ClassPool pool, ClassLoader classLoader, String className, ClassLoadingChecker classLoadingChecker) throws NotFoundException, IOException, CannotCompileException, IllegalArgumentException, IllegalAccessException, InvocationTargetException {
         Class<?> c = null;
         try {
-            c = classLoader.loadClass(className);
+            // Using different approach to load classes via classloader
+            // c = classLoader.loadClass(className);
+            c = Class.forName(className, false, classLoader);
```

Comment:
Hi @emeroad ,

Thats good news, our approach while working for Jboss plugin was to go for minimum code change in existing Pinpoint project.
So, we choosed to solve the logger problem via defining logmanager property and logmanager jar in Jboss configuration.

Regarding Classloading issue, will be happy to be involve in the discussion and would like to contribute as well, if a more better approach from above code change in PlainClassLoaderHandler can be done.

Thanks